packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -58,7 +58,11 @@ export class PipelineService {
    * to run the pipeline nodes.
    */
   static async getRuntimeImages(): Promise<any> {
-    const runtimeImages = await FrontendServices.getMetadata('runtime-images');
+    let runtimeImages = await FrontendServices.getMetadata('runtime-images');
+
+    runtimeImages = runtimeImages.sort(
+      (a: any, b: any) => 0 - (a.name > b.name ? -1 : 1)
```

Comment:
why the `0 -`? why not just swap the `1` and `-1` in the ternary?

Reply:
Just an easy way to accommodate `<`, `>` and `==`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort